.buildkite/pipeline.yml

```diff
@@ -27,9 +27,9 @@ steps:
     matrix:
       setup:
         python:
+          - '3.11'
           - '3.10'
           - '3.9'
-          - '3.8'
```

Comment:
Are you dropping Python 3.8 support on purpose? If yes we need to update `python_requires` in setup.py.

Reply:
I thought Python 3.8 could be removed from CI but as it is still supported that would be a silly thing to do. I've reverted this change